stratum/hal/lib/common/utils.cc

```diff
@@ -333,6 +333,14 @@ std::string MacAddressToYangString(
                          mac_address & 0xFF);
 }
 
+::google::protobuf::uint64 YangStringToMacAddress(
+    const std::string& yang_string) {
+  std::string tmp_str = yang_string;
+  // Remove colons
+  tmp_str.erase(std::remove(tmp_str.begin(), tmp_str.end(), ':'), tmp_str.end());
+  return strtoul(tmp_str.c_str(), NULL, 16);
```

Comment:
I'd suggest using `strtoull` since `unsigned long` can be 32-bit in some cases.
https://en.cppreference.com/w/cpp/language/types

Reply:
Got it, thanks.